File: TrabalhoFinal [Redes]/src/br/quixada/ufc/rsd/Server/MotoTaxi.java
package br.quixada.ufc.rsd.Server;

public class MotoTaxi {
	
	private static int id_motoTaxi; 

	private String nome; 
	private String telefone;
	private String praca;
	
	
	public MotoTaxi (String nome, String telefone, String praca) {
		this.id_motoTaxi++;
		
		this.nome = nome;
		this.telefone = telefone; 
		this.praca = praca;
	}


	public static int getId_motoTaxi() {
		return id_motoTaxi;
	}

	public String getNome() {
		return nome;
	}


	public void setNome(String nome) {
		this.nome = nome;
	}


	public String getTelefone() {
		return telefone;
	}


	public void setTelefone(String telefone) {
		this.telefone = telefone;
	}


	public String getPraca() {
		return praca;
	}


	public void setPraca(String praca) {
		this.praca = praca;
	}

}

File: ClientePy/entity/Mototaxi.py
class Mototaxi:
    def __init__(self,id_mototaxi, nome, telefone, praca):
        self.__id_mototaxi = id_mototaxi
        self.__nome = nome
        self.__telefone = telefone
        self.__praca = praca

    def getId_mototaxi(self):
        return self.__id_mototaxi

    def setId_mototaxi(self, id_mototaxi):
        self.__id_mototaxi = id_mototaxi

    def getNome(self):
        return self.__nome

    def setNome(self, nome):
        self.__nome = nome

    def getTelefone(self):
        return self.__telefone

    def setTelefone(self, telefone):
        self.__telefone = telefone

    def getPraca(self):
        return self.__praca

    def setPraca(self, praca):
        self.__praca = praca